libclc/opencl/include/clc/opencl/integer/bit_reverse.h

```diff
@@ -0,0 +1,21 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __CLC_OPENCL_INTEGER_BIT_REVERSE_H__
```

Comment:
I think these 4 OpenCL decls should also have  `#ifdef cl_khr_extended_bit_ops`.

Reply:
done, thanks